public/app/percona/settings/components/Platform/Platform.messages.ts

```diff
@@ -0,0 +1,8 @@
+export const Messages = {
+  title: 'Connect PMM to Percona Portal',
+  connectSucceeded: 'Successfully connected PMM to Percona Portal',
+  pmmServerName: 'PMM Server Name *',
```

Comment:
if we add `required` to TextInputField, doesn't `*` get appended automatically?

Reply:
Yes, forgot about it, done!